src/Compilers/CSharp/Portable/Symbols/ExtraAnnotations.cs

```diff
@@ -15,8 +15,9 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     internal enum AttributeAnnotations
     {
         None = 0,
-        NotNullWhenFalse,
-        EnsuresNotNull,
+        NotNullWhenTrue = 1 << 0,
+        NotNullWhenFalse = 1 << 1,
+        EnsuresNotNull = NotNullWhenTrue | NotNullWhenFalse,
```

Comment:
>NotNullWhenTrue | NotNullWhenFalse [](start = 25, length = 34)

It feels strange to have this flags for a method that doesn't return a bool.

Reply:
I think it's ok for our internal representation to be suited for our usage (ie. NullableWalker). I prefer to leave this as-is.

---
In reply to: [191564968](https://github.com/dotnet/roslyn/pull/26828#discussion_r191564968) [](ancestors = 191564968)